src/main/java/org/apache/maven/shared/artifact/filter/ScopeArtifactFilter.java

```diff
@@ -25,7 +25,7 @@
 import org.apache.maven.artifact.Artifact;
 import org.apache.maven.artifact.DefaultArtifact;
 import org.apache.maven.artifact.resolver.filter.ArtifactFilter;
-import org.codehaus.plexus.logging.Logger;
+import org.slf4j.Logger;
```

Comment:
can this change be separated into its own PR?

Reply:
https://github.com/apache/maven-common-artifact-filters/pull/18